bindings/rascal/representations/base.py

```diff
@@ -26,3 +26,49 @@ def CalculatorFactory(rep_options):
              'The available combinations are: {}').format(
                 name, list(_representations.keys())))
     return _representations[name](*rep_options['args'])
+
+def cutoff_function_dict_switch(cutoff_function_type, **kwargs):
+    """
+    return appropriate dict for the cutoff function parameters
+    """
+    if cutoff_function_type == 'Cosine':
```

Comment:
Let's call this 'ShiftedCosine' for compatibility with B-P

Reply:
```suggestion
    if cutoff_function_type == 'ShiftedCosine':
```

Reply:
this renaming is outside of the aim of this branch :)

Reply:
Is it though? When do we do the renaming then? Because this seems like a pretty good place, then we don't have the extra pain wrangling names when B-P is merged in.

Reply:
I count 14 files where the renaming would need to be done (excluding the kernel with exactly the same name, "Cosine", which I think is slightly misleading -- it's a dot product -- but that's a discussion for another day):

```
bindings/rascal/representations/spherical_covariants.py
65:                 cutoff_function_type="Cosine", normalize=True,

bindings/rascal/representations/spherical_expansion.py
54:                 gaussian_sigma_constant=0., cutoff_function_type="Cosine",

bindings/rascal/representations/spherical_invariants.py
63:                 cutoff_function_type="Cosine",

examples/spherical_invariants_example.cc
72:  json fc_hypers{{"type", "Cosine"},

examples/test_nl.cc
99:  json fc_hypers{{"type", "Cosine"},
143:      {"zeta", 2}, {"target_type", "Structure"}, {"name", "Cosine"}};

examples/spherical_expansion_example.cc
72:  json fc_hypers{{"type", "Cosine"},

examples/spherical_invariants_profile.cc
77:  json fc_hypers{{"type", "Cosine"},

examples/spherical_expansion_profile.cc
74:  json fc_hypers{{"type", "Cosine"},

src/representations/calculator_spherical_expansion.hh
676:      if (fc_type.compare("Cosine") == 0) {
677:        this->cutoff_function_type = CutoffFunctionType::Cosine;
679:            make_cutoff_function<CutoffFunctionType::Cosine>(fc_hypers);
683:                               ": \'Cosine\'.");
796:    case CutoffFunctionType::Cosine: {
797:      this->compute_by_radial_contribution<CutoffFunctionType::Cosine>(

src/representations/cutoff_functions.hh
47:    enum class CutoffFunctionType { Cosine, RadialScaling, End_ };
82:    struct CutoffFunction<internal::CutoffFunctionType::Cosine>

tests/test_calculator.hh
123:        {{"type", "Cosine"},
126:        {{"type", "Cosine"},
191:        {{"type", "Cosine"},
194:        {{"type", "Cosine"},
269:        {{"type", "Cosine"},
272:        {{"type", "Cosine"},
352:        {{"type", "Cosine"},
355:        {{"type", "Cosine"},
441:        {{"type", "Cosine"},
478:        {{"type", "Cosine"},

scripts/generate_spherical_covariant_ref_data.py
77:                          "cutoff_function_type": "Cosine",

scripts/generate_spherical_invariant_ref_data.py
86:                                    "cutoff_function_type": "Cosine",

scripts/generate_spherical_expansion_ref_data.py
77:                                          "cutoff_function_type": "Cosine",
```
I'd say this is 15 minutes of work at most; if you agree, I'll go ahead and do the renaming.

Reply:
If you feel like it just go on with the renaming of the cutoff function. Note that you need to generate again the reference data that have this keyword so that the test can work properly.

Reply:
Yeah, I noticed after the tests failed. Just this simple rename resulted in ~16 MiB of changes... I'm really not a fan of these huge binary "reference data" files (which really just compare to previous versions).